man/systemd.xml

```diff
@@ -392,6 +392,25 @@
     <citerefentry><refentrytitle>systemd.special</refentrytitle><manvolnum>7</manvolnum></citerefentry>
     for details about these target units.</para>
 
+    <para>systemd only keeps a minimal set of units loaded into memory. Specifically, only units are kept loaded into
```

Comment:
"Specifically, the only units that are kept loaded into memory are those for which at least one of the following conditions is true"